MyHeartCounts/Modules/DeferredConfigLoading.swift

```diff
@@ -213,6 +213,7 @@ enum DeferredConfigLoading {
                 }
                 baseModules
                 NewsManager()
+                TimeZoneTracking()
```

Comment:
Don't we also need to register the account key in the account configuration?

Reply:
good catch; i've updated the AccountConfiguration accordingly